src/routes/TransactionsList.tsx

```diff
@@ -3,14 +3,43 @@ import Tabs from 'components/Tabs';
 import Transactions from 'components/Transactions/Transactions';
 import { Box, Grid, InputAdornment, MenuItem, Stack, TextField, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
-
-// TO BE REMOVED
-import { dummyTransactionsData } from '../stories/utils/mocks';
 import { Search } from '@mui/icons-material';
 import { theme } from '@pagopa/mui-italia';
+import { AxiosResponse } from 'axios';
+import { useLoaderData, useNavigate } from 'react-router-dom';
+import utils from 'utils';
+import { TransactionsResponse } from '../../generated/apiClient';
+import { ArcRoutes } from './routes';
 
 export default function TransactionsList() {
   const { t } = useTranslation();
+  const transactions = useLoaderData();
+  const navigate = useNavigate();
+
+  const rows = utils.converters.prepareRowsData({
+    transactions: (transactions as AxiosResponse<TransactionsResponse>).data,
+    status: { label: t('app.transactions.payed') },
+    payee: { multi: t('app.transactions.multiEntities') },
+    action: (id) => navigate(`${ArcRoutes.TRANSACTION}`.replace(':ID', id))
+  });
+  const paidByMe = utils.converters.prepareRowsData({
+    transactions: (transactions as AxiosResponse<TransactionsResponse>).data.filter(
+      (t) => t.payedByMe == true
```

Comment:
tip: `({payedByMe}) => payedByMe` is more concise and elegant way to do this :)

maybe a more efficient approach here is to remove the rows computation and just use paidByMe and ownedByMe arrays to obtain a new 'all items' array

Reply:
I'm sorry, I don't get what you mean. The all items array is the one we get from the API, so there's no need to use the other ones to obtain it.